test/cache-option.test.js

```diff
@@ -27,303 +27,307 @@ const otherOtherOtherCacheDir = findCacheDir({
 
 jest.setTimeout(30000);
 
-describe('cache option', () => {
-  let compiler;
-
-  beforeEach(() => {
-    compiler = getCompiler({
-      entry: {
-        one: `${__dirname}/fixtures/cache.js`,
-        two: `${__dirname}/fixtures/cache-1.js`,
-        three: `${__dirname}/fixtures/cache-2.js`,
-        four: `${__dirname}/fixtures/cache-3.js`,
-        five: `${__dirname}/fixtures/cache-4.js`,
-      },
-    });
+if (getCompiler.isWebpack4()) {
```

Comment:
Let's keep testing `cache` here for webpack@4 and webpack@5, and fix cache tests in PR where we implement new cache API for webpack@5